index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+
+    <div class="starting-page">
+        <header class="starting-page>
+            <nav id="navigator_bar">
+                <a class="logo" href="#"><h2>Mylogo</h2></a>
+                <nav class="navigator_menu"><img src="assets/Enabled.png">
+                </nav>
+            </nav>
+        </header>
```

Comment:
- [ ] [Optional] You should stick to the design as much as possible. Please align the logo on the left of the page and the hamburger button on right. Also please remove the underline for a logo to stick to the design. 
- [ ] [Optional] I would suggest you please use the background image, you can do it by downloading the whole background image and using it.
### Your design
![lyour](https://user-images.githubusercontent.com/81727988/219388951-46b60c27-b67b-4a71-ac50-e52a3c6f93f6.png)


### Figma design
![figma](https://user-images.githubusercontent.com/81727988/219388979-a3789f0a-4105-47ca-8c2b-78577fc79ff1.png)

Reply:
Resolved the issue, thank you for notifying me.